src/main/webapp/web.xml

```diff
@@ -7,8 +7,88 @@
         <servlet-name>index</servlet-name>
         <servlet-class>mate.controller.IndexController</servlet-class>
     </servlet>
+    <servlet>
```

Comment:
it would be better to write servlet-mapping just after each servlet and check your all web.xml in the `Good URLs naming in your project` in the common mistakes